src/main/java/io/vinyldns/java/model/membership/Group.java

```diff
@@ -0,0 +1,87 @@
+package io.vinyldns.java.model.membership;
+
+import java.util.Set;
+import org.joda.time.DateTime;
+
+public class Group {
+    private final String name, email, id;
+    private String description;
```

Comment:
I would leave`description` as `final` and just explicitly set it to `null` in the other constrctor